blueocean-personalization/src/main/js/components/ActionLink.jsx

```diff
@@ -0,0 +1,7 @@
+import React, { Component } from 'react';
+
+export default class ActionLink extends Component {
+    render() {
+        return <div>Test</div>;
```

Comment:
You're right, not good to litter Test text all over the main dashboard, even for a day or two until the other tickets get completed. For now I updated it just to write a HTML comment in there so I can still easily prove / test the integration (see commit 01ffc01). Will that work?